trymito.io/components/CTAButtons/CTAButtons.tsx

```diff
@@ -38,6 +45,25 @@ const CTAButtons = (props: {
                     href="mailto:founders@sagacollab.com"
                 />
             }
+            {props.variant === 'book a demo' && 
+                <>
+                    <div className='only-on-desktop'>
+                        <TextButton 
+                            text='Book an Enterprise Demo'
+                            href="https://calendly.com/jake_from_mito/30min"
+                            variant='secondary'
+                        />
+                    </div>
+                    <div className='only-on-mobile'>
+                        <TextButton 
+                            text='Book a Demo'
+                            href="https://calendly.com/jake_from_mito/30min"
+                            variant='secondary'
+                        />
```

Comment:
Can we just:
1. Move the Cleanly link to a constant in the codebase so we can make sure we update it in all the right places
2. Add a plausible class here so we can see the effectiveness of this. There is documentation here: https://github.com/mito-ds/mito/blob/main/trymito.io/utils/plausible.tsx

It looks like that file doesn't exist on this branch because I didn't merge my PR into dev, just straight to main so we could release it. Let me know if that causes merge conflicts and I can try to fix it up. 

Now that we're having multiple people working on the website maybe we want to start going through dev again? Lmk what you want here.